### PR TITLE
SWESARR Tutorial Update

### PR DESCRIPTION
**Pull request by drboyd**
Updated SWESARR Tutorial. Includes comparison to SWE values provided by SnowExSQL and a few extra visualizations for comparing SWE and SWESARR data.

The default RAM allocation (2GB) for CryoInTheCloud can't handle the final figure generated by this notebook. This is poor memory management on my part. I've commented out the figure for the tutorial. Currently, this notebook uses roughly 1GB of RAM.

Most libraries are contained by default via CryoInTheCloud but SnowExSQL isn't. This may be an issue with merging to the main branch.

### CODE REVIEW

**Comment by drboyd** [comment]
Hi @scottyhq, sorry for the very, very late contribution from me. 
Looks like the following error is keeping me from merging:
```
/home/runner/work/website-2024/website-2024/book/tutorials/swesarr/swesarr_tut.ipynb:530003: 
WARNING: 'myst' cross-reference target not found: 'Table-of-Contents' [myst.xref_missing]
```

**Comment by drboyd** [comment]
Hi @scottyhq 
Website [seems to render fine.](https://deploy-preview-15--snowex-website-2024.netlify.app/tutorials/swesarr/swesarr_tut) I think the errors are related to hvplot rendering:
`WARNING: skipping unknown output mime type: application/vnd.holoviews_load.v0+json`

All the same, figures are generated A-ok for this tutorial. 

For my laptop, the figures could do with some shrinking in the width to fit more cleanly on the website page. Other than that, I don't think there's any issues.

**Comment by scottyhq** [comment]
> The default RAM allocation (2GB) for CryoInTheCloud can't handle the final figure generated by this notebook.  

Feel free to show people how to launch a larger server during your tutorial! 

image

> I think the errors are related to hvplot rendering:
WARNING: skipping unknown output mime type: application/vnd.holoviews_load.v0+json

#16 should've fixed this

> Most libraries are contained by default via CryoInTheCloud but SnowExSQL isn't.

We plan to update the default image to include this and other updated libraries today or tomorrow (https://github.com/CryoInTheCloud/hub-image/pull/125).

File: book/tutorials/swesarr/swesarr_tut.ipynb [hunk 1]
```diff
@@ -0,0 +1,840 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "56f3be28",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "![NASA](http://www.nasa.gov/sites/all/themes/custom/nasatwo/images/nasa-logo.svg)\n",
```
Comment (scottyhq) [inline book/tutorials/swesarr/swesarr_tut.ipynb:24]: This SVG isn't rendering for some reason on cryocloud

File: book/tutorials/swesarr/swesarr_tut.ipynb [hunk 2]
```diff
@@ -0,0 +1,840 @@
...
+    "import datashader as ds           # https://www.lfd.uci.edu/~gohlke/pythonlibs/\n",
+    "import hvplot.xarray              # !conda install hvplot\n",
+    "import hvplot.pandas\n",
+    "import holoviews as hv\n",
+    "\n",
+    "# append the subfolders of the current working directory to pythons path\n",
+    "import os, sys, glob\n",
+    "\n",
+    "swesarr_subdirs = [\"data\", \"util\"]\n",
+    "tmp = [sys.path.append(os.getcwd() + \"/\" + sd) for sd in swesarr_subdirs]\n",
```
Comment (scottyhq) [inline book/tutorials/swesarr/swesarr_tut.ipynb:262]: for tutorial data on cryocloud you'll likely have better performance downloading to `/tmp` (https://docs.2i2c.org/user/topics/data/)

**Review by scottyhq — APPROVED** [review]
thanks @drboyd left some recommendations but feel free to merge whenever

**Comment by drboyd** [comment]
Thanks for catching these errors, @scottyhq. I'll address these suggestions, probably tweak the size of the figures so they fit more cleanly on the website, and merge on through tomorrow. Thanks especially for the tips here and catching that dead link I glanced over.

**Comment by drboyd** [comment]
Alright, I think that's everything. Thank you so much, @scottyhq !

**Pull request merged**